### ISSUE TITLE
Reading age calculations

### ISSUE THREAD

**Issue by ldawson13** [NewcastleRSE/nhs_readability_react#37]
Could this be calculated using the FKGL instead? The reading age is essentially the FKRG score +5. It’s probably easier to do it in whole numbers rather than 10-11, 11-12 so I’ve put a column of whole numbers down the side. Not sure if this picture has pulled through.

![image](https://user-images.githubusercontent.com/116188097/203985405-2a76d7c6-5310-4138-97d2-da2be19beca2.png)

**Comment by Darren404** [comment]
I put your test text into the deployment at https://readability.ncldata.dev/ and it scored 15.4 yrs
Was the change discussed above cancelled out by a subsequent change?
Hemmingway Editor gives the same text a grade of 9, which is 14 yrs.
Word (going from pure FK) gives it 8.2, which is about 13.
Smog tests give it about 15.

**Comment by ldawson13** [comment]
Hi Darren, I'm not sure whether the changes have been cancelled out (I'm from the health literacy side of things, not a programmer) but ideally we'd want the score to be the word FK grade converted to age in years please. So for this text, we'd want it to say 13 yrs or 13.4 yrs (based on the pure FK grade of 8.2). Thank you!

**Comment by Darren404** [comment]
Hi Lorna,
I'm not part of the project team and only found this repository when we discovered a deployment of the in-development tool was being used to assess some GP Websites against the age requirements.
I've raised it directly with one of the developers, because it's important that the calculation is accurate if it is being used in production. Our tests show it consistently inflating the reading age over the Word FK and Hemmingway editor scores.
